thewalrus/tests/test_csamples.py

```diff
@@ -65,3 +69,29 @@ def test_dist_thermal():
     rel_freq = freq / n_samples
     expected = (1 / (1 + n_mean)) * (n_mean / (1 + n_mean)) ** (np.arange(len(rel_freq)))
     assert np.allclose(rel_freq, expected, atol=10 / np.sqrt(n_samples))
+
+
+@pytest.mark.parametrize("nmodes", [3, 4, 5, 6])
+def test_number_moments_multimode_thermal(nmodes):
+    r"""Test the correct values of the photon number means and covariances"""
+    assert nmodes > 1
```

Comment:
This is redundant, since the number of modes is hardcoded into the test :laughing:

Reply:
No, it is not. If you try to set nmodes=1  you will get an error. np.cov will return a two dimensional array if your data is multidimensional but seems like it returns a scalar if the data is one dimensional hence the assert, to remind myself to not run it with nmodes=1

Reply:
Ah, right, what I meant was, that the test is being run explicitly with `nmodes in [3, 4, 5, 6]`, so this line will always pass.